### DISCUSSION TITLE
Homekit Type/Icon for Washer/Dryer

### DISCUSSION THREAD

**Discussion by asoie** [nVuln/homebridge-lg-thinq#120] [General]
Just installed and configured my washer/dryer.  They show up in the Home app as simple faucets.  Is this the expected behavior?

**Comment by nVuln** [comment]
yup, homekit do not support washer at this time, so we simulate it as water faucet (same for dishwasher and dryer)

**Comment by Mayo99** [comment]
Any update to the ability to updating HomeKit ICONs for Washer, Pedestal Washer, and or Dryer

**Reply by nVuln** [reply]
no, atm homekit still not update Washer/Dryer
